include/TPP/Dialect/LinalgX/TransformOps/LinalgXTransformOps.td

```diff
@@ -207,4 +207,39 @@ def MapLinalgToTppOp : Op<Transform_Dialect, "structured.map_linalg_to_tpp", [
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// FoldUnitExtentDimsOp
```

Comment:
I don't understand why this is in `LinalgX` and not on its own dialect. While this operates on `linalg` ops, it's not an extension to the `linalg` dialect.